test/C/src/federated/transient/TransientDownstreamWithTwoUpstream.lf

```diff
@@ -0,0 +1,125 @@
+/**
+ * This LF program tests if a transient federate corretly leaves then joins the federation. It also
+ * tests if the transient's downstream executes as expected, that is it received correct TAGs,
+ * regardless of the transient being absent or present. In this test:
+ * - the transient federate spontaneously leaves the federation after 2 reactions to input port in,
+ * - the downstream of the transient federate has one persistent and one transient upstreams.
+ */
+target C {
+  timeout: 2 s
+}
+
+import Up from "TransientDownstreamWithTimer.lf"
+import Middle from "TransientDownstreamWithTimer.lf"
+
+preamble {=
+  #include <stdlib.h>
+  #include <string.h>
+=}
+
+/** Persistent federate that is responsible for lauching the transient federate */
+reactor TransientExec(offset: time = 0, period: time = 0, fed_instance_name: char* = "instance") {
```

Comment:
Same comment about `offset` being renamed `launch_time` and `period` being removed.